### PR TITLE
layer2 store

### PR DESCRIPTION
**Pull request by blockchain-develop**

### CODE REVIEW

**Comment by laizy** [comment]
where is the proof verification code?

**Comment by blockchain-develop** [comment]
> where is the proof verification code?

fixed

File: client/client.go [hunk 1]
```diff
@@ -364,6 +365,25 @@ func (this *ClientMgr) PreExecTransaction(mutTx *types.MutableTransaction) (*sdk
 	return preResult, nil
 }
 
+func (this *ClientMgr) VerifyStoreProof(key []byte, value []byte, proof []byte, stateRoot []byte) (bool, error) {
```
Comment (laizy) [inline client/client.go:368]: please add unit testcase.

Comment (blockchain-develop) [reply client/client.go:368]: fixed

**Pull request merged**